src/index.css

```diff
@@ -69,8 +72,45 @@
     -moz-appearance: textfield; /* Firefox */
   }
   input[type='checkbox'] {
-    accent-color: var(--primary);
-  }  
+    -webkit-appearance: none;
+    position: relative;
+    border-radius: 4px;
+  }
+  input[type='checkbox']:not(:checked) {
+    border: 1px solid rgb(255 255 255 / 0.6);
+  }
+  input[type='checkbox']:not(:checked):hover {
+    border-color: rgb(255 255 255 / 0.8);
+  }
+  input[type='checkbox']:checked {
+    border: 0px;
+    color: rgba(22, 22, 22, var(--tw-text-opacity));
+    background-image: linear-gradient(
+      90deg,
+      var(--gradientFirst) 0%,
+      var(--gradientMiddle) 100%
```

Comment:
You don't want the gradientLast here ?

Reply:
`gradientLast` is used for the hover effect in the buttons and using gradientFirst to gradientMiddle makes the checkbox background look just like the buttons' background